test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/BuildIncrementalismTest.cs

```diff
@@ -56,5 +56,34 @@ public class BuildIncrementalismTest : MSBuildIntegrationTestBase
                 }
             }
         }
+
+        [Fact]
+        [InitializeTestProject("SimpleMvc")]
+        public async Task Build_RegeneratesTagHelperInputs_IfFileChanges()
```

Comment:
Still prefer naming this `RazorGenerate_Rebuilds_IfTagHelpersChanged` to make it consistent with others.

Reply:
Sure